cli/server/server.go

```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/kelseyhightower/envconfig"
 )
 
+var version = "dev"
```

Comment:
What do you think, if instead of having it hard-coded we replace it at the build time with something like `git describe` using `-ldflags`

Reply:
https://github.com/src-d/ci/blob/v1/Makefile.main#L56

this one is default value when you do `go run` for example.

Reply:
Suggestion/request: to make it easier for new team members, it would be nice to have your explanation as a comment above the variable, like:
`version will be replaced automatically by the CI build. See <link>`

Reply:
oh come on. it's how all src-d go projects work:

https://github.com/src-d/enry/blob/master/cli/enry/main.go#L20
https://github.com/src-d/borges/blob/master/cli/borges/main.go#L19
https://github.com/src-d/go-git/blob/master/cli/go-git/version.go#L5
https://github.com/bblfsh/bblfshd/blob/master/cli/bblfshd/main.go#L20
https://github.com/bblfsh/dashboard/blob/master/server/cmd/bblfsh-dashboard/main.go#L20

Reply:
Huge 👍 on what @carlosms says.

@smacker thank you for the links!

Caring about new contributors, including src-d team members is very aligned with https://github.com/src-d/code-annotation/blob/master/CODE_OF_CONDUCT.md#our-standards

> Showing empathy towards other community members

Reply:
put comments about CI in the __code__? really?
what next? describe our kubernetes configuration in the code?
comment that algorithm is going to work fast because we run it on science3 with GPU?

whatever, I added a comment